TheMovieDb/Home/Utils/MovieGroupSections.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  GroupSections.swift
+//  TheMovieDb
+//
+//  Created by Javier Cueto on 30/10/21.
+//
+
+import Foundation
+
+enum MovieGroupSections: Int {
+    case popular
+    case trending
+    case playingNow
+    case topRated
+    case upcoming
+    
+    var description: String {
+        switch self {
+        case .popular: return "Popular"
+        case .trending: return "Trending"
+        case .playingNow: return "Playing now"
+        case .topRated: return "Top rated"
+        case .upcoming: return "Upcoming"
+        }
+    }
+}
```

Comment:
Can we have a more robust model for group sections?

Reply:
Currently  I´m exploring new ways